### ISSUE TITLE
Repository is missing tags

### ISSUE THREAD

**Issue by thvd** [vlocityinc/vlocity_build#689]
So till 1.17.6 there are tags created for all the releases: https://github.com/vlocityinc/vlocity_build/tags

But the 1.17.7 and 1.17.8 are missing, see releases here on npmjs.org: https://www.npmjs.com/package/vlocity?activeTab=versions

**Comment by pawel-id** [comment]
The [Release](https://github.com/vlocityinc/vlocity_build/releases) section of the repository is also missing updates. Currently, the latest release available is version 1.17.6 from December. However, newer [versions have been published on the npm registry](https://www.npmjs.com/package/vlocity?activeTab=versions). It is good practice to keep the repository's Releases section aligned with the versions published on the npm registry. Adding git tags (as @thvd suggested) and corresponding Releases would be appropriate.

The lack of this information can create uncertainty for Vlocity customers regarding the correct CLI version to use. Should they follow the repository's version 1.17.6 or the npm registry's version 1.17.8?

**Comment by thvd** [comment]
> The lack of this information can create uncertainty for Vlocity customers regarding the correct CLI version to use. Should they follow the repository's version 1.17.6 or the npm registry's version 1.17.8?

I agree to this and how do consumers know what's changed if the releases aren't tagged?
